source/extensions/transport_sockets/tls/ssl_socket.cc

```diff
@@ -288,7 +295,54 @@ void SslSocket::onConnected() { ASSERT(info_->state() == Ssl::SocketState::PreHa
 
 Ssl::ConnectionInfoConstSharedPtr SslSocket::ssl() const { return info_; }
 
+static void keylogCallback(const SSL* ssl, const char* line) {
```

Comment:
Can we implement this callback in `SslSocket()` and remove static?

Reply:
It seems that this callback is defined by the openssl lib function SSL_CTX_set_keylog_callback().

Reply:
You may define this callback like [this](https://github.com/envoyproxy/envoy/blob/main/source/extensions/transport_sockets/tls/context_impl.cc#L161). `verifyCallback()` is defined as class method.